server/followers/views.py

```diff
@@ -150,7 +159,8 @@ def retrieve(self, request, *args, **kwargs):
                 status=status.HTTP_404_NOT_FOUND)
 
         try:
-            remote_follower = instance.values('remoteFollowers')[0]['remoteFollowers']
+            remote_follower = instance.values('remoteFollowers')[
+                0]['remoteFollowers']
```

Comment:
Python allows you to do this? I don't understand the change but okay.

Reply:
my built in python prettier did this for me, but the code works